### ISSUE TITLE
Broken pipe in websocket server

### ISSUE THREAD

**Issue by JWalker1995** [denoland/deno#19959]
First of all thank you for this amazing project!

I'm trying to narrow down a problem I'm experiencing with a tls https+wss server using Deno.serve and Deno.upgradeWebSocket. At some point, the message event stops being called and soonafter the process crashes with this uncatchable error:
```
error: Uncaught (in promise) TypeError: Broken pipe (os error 32)
    at ext:deno_websocket/01_websocket.js:506:17
    at Object.action (ext:deno_web/02_timers.js:153:11)
    at handleTimerMacrotask (ext:deno_web/02_timers.js:67:10)
    at eventLoopTick (ext:core/01_core.js:189:21)
```

Still trying to create a minimally reproducible example.

```
> deno --version
deno 1.35.3 (release, x86_64-unknown-linux-gnu)
v8 11.6.189.12
typescript 5.1.6
```

**Comment by milaz** [comment]
I suspect that the client terminates the connection abruptly.

Even so, should not lead to the server crash. Otherwise, it would be a "great" attack vector on Deno servers: connect to WebSocket, block packets, Deno crashes in 2 minutes.

Please, see #16676 for more.

**Comment by littledivy** [comment]
Hi, This was fixed in the latest release (1.36.1). Feel free to reopen if its still happening.
